python/lsst/jointcal/jointcal.py

```diff
@@ -341,6 +343,10 @@ def run(self, dataRefs, profile_jointcal=False):
         with pipeBase.cmdLineTask.profile(load_cat_prof_file):
             self._write_results(associations, astrometry.model, photometry.model, visit_ccd_to_dataRef)
 
+        # HACK convert to unitless quantities
```

Comment:
I'm not sure this is a hack, it's just what it is. We could also put in the dimensionless part in each measurement (though then they'd have to go in the tests).

Oh! why don't you make that dict comp include an `if type(val) is not astropy.Quantity` or something, for if we add unitted values in the future?